twitter/searcher.py

```diff
@@ -0,0 +1,182 @@
+import tweepy
+from typing import Dict, List, Any, Set, Tuple
+from datetime import datetime
+import json
+
+# from boto3 import client as boto_client
+
+from common.data_classes import RawSubmission
+from common.enums import DataSource
+
+from clients.laravel_client import bulk_upload_submissions
+from common.config import (
+    TWITTER_LAST_RUN_FILENAME,
+    # TWITTER_LAST_RUN_BUCKET,
+    TWITTER_LARAVEL_API_KEY,
+    TWITTER_CONSUMER_KEY,
+    TWITTER_CONSUMER_SECRET,
+    TWITTER_ACCESS_TOKEN,
+    TWITTER_ACCESS_TOKEN_SECRET,
+)
+
+# from common.utils import write_raw_submissions_to_csv
+
+QUERIES = [
+    # recommended by freezman
+    "#bluefall",
+    "#PoliceBrutality",
+    "#PoliceBrutailtyPandemic",
+    "#Protests2020",
+    # cities by population - protests
+    "#nycProtests",
+    "#newyorkProtests",
+    "#losangelesProtests",
+    "#laprotests",
+    "#stLouisProtests",
+    "#stlProtests",
+    "#philadelphiaprotests",
+    "#phillyprotests",
+    "#chicagoprotests",
+    "#houstonprotests",
+    "#pheonixProtests",
+    "#miamiProtests",
+    "#dcprotests",
+    "#WashingtonDCProtest",
+    "#seattleprotest",
+    "#austinprotest"
+    # found from 949mac's endpoint https://api.846policebrutality.com/api/incidents?include=evidence
+    "#GeorgeFloyd",
+    "#JusticeForGeorgeFloyd",
+    "#AbolishThePolice",
+    "#BlackLivesMatter",
+    "tear gas",
```

Comment:
A little concerned about how weird things are gonna get with a non-hash tag serach but 🤷

Reply:
we can pull it if we see a lot of garbage

Reply:
next step would be to create a UI so we could add / remove search terms